File: Baekjoon/문자열/11655.py
# https://www.acmicpc.net/problem/11655

import sys
input = sys.stdin.readline

s = input().rstrip()
result = []

for c in s:
    if c.islower():
        result.append(chr((ord(c) - ord('a') + 13) % 26 + ord('a')))
    elif c.isupper():
        result.append(chr((ord(c) - ord('A') + 13) % 26 + ord('A')))
    else:
        result.append(c)
        
print("".join(result))

File: Baekjoon/문자열/11655.cpp
// https://www.acmicpc.net/problem/11655

#include <iostream>
#include <string>
using namespace std;

int main() {
    string s;
    getline(cin, s);
    for (auto &c : s) {
        if ('a' <= c && c <= 'z') {
            c = (c - 'a' + 13) % 26 + 'a';
        } else if ('A' <= c && c <= 'Z') {
            c = (c - 'A' + 13) % 26 + 'A';
        }
    }
    cout << s << '\n';
    return 0;
}